src/Results.js

```diff
@@ -0,0 +1,54 @@
+import {useState, useEffect} from 'react';
+import Movie from './Movie';
+import People from './People';
+
```

Comment:
Really great use of components all cross your app. This was definitely something I found to be quite challenging, so its amazing to see your implementation.